test/adapter/json_api/resource_meta_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi
+        class ResourceMetaTest < Minitest::Test
+          class MetaPostSerializer < ActiveModel::Serializer
+            attributes :id
+            meta do
+              { stuff: "value" }
```

Comment:
Would be great to make a test using the block syntax (like here) to make use of some of `object`'s property, like
```ruby
meta do
  { comments_count: object.comments.count }
end
```
and an other test that makes use of the direct syntax (no bloc) as follows:
```ruby
meta { stuff: 'value' }
```

Detail: use single quotes for strings without interpolation, otherwise Rubocop will complain when we try to merge it.